### ISSUE TITLE
Missing Nodes after updating

### ISSUE THREAD

**Issue by Rojinski** [FizzleDorf/AIT#63]
Hello everyone,

After updating AIT nodes, I can't use my workflows using it. Everything is up-to-date (ComfyUi and nodes). I've tried to use ComfyUI Manager to download missing nodes and it just shows that, of course, AIT is installed.  I've used the "git pull" command to be sure but it says is already up-to-date. I attach a screenshot of what it says. I hope someone can help me because I use these workflows a lot. :) And If I'm doing something wrong (but I'm used to update the nodes), I'd like to know and I'm sorry on advance to be annoying... 

Have a beautiful week-end ! :)

![Screenshot 2023-10-06 at 15-14-46 ComfyUI](https://github.com/FizzleDorf/AIT/assets/122015375/81a7831a-5752-4102-932b-1666bdfce4a6)

**Comment by FizzleDorf** [comment]
Is this the case when you clone manually? This issue happens sometimes when installing through the manager

**Comment by Rojinski** [comment]
> Is this the case when you clone manually? This issue happens sometimes when installing with the manager

Yes, I've tried to clone it (git clone) and it's still the same.

**Comment by MaxLanc** [comment]
Can confirm, same bug here
